R/bundlePython.R

```diff
@@ -0,0 +1,134 @@
+# Create anonymous function that we can later call to get all needed python
+# metdata for the manifest
+pythonConfigurator <- function(python,
+                               forceGenerate = FALSE,
+                               condaMode = FALSE) {
+
+  if (is.null(python)) {
+    return(NULL)
+  }
+
+  force(condaMode)
+  force(forceGenerate)
+
+  function(appDir) {
+    withCallingHandlers(
+      inferPythonEnv(
+        appDir,
+        python = python,
+        condaMode = condaMode,
+        forceGenerate = forceGenerate
+      ),
+      error = function(err) {
+        cli::cli_abort(
+          "Failed to detect python environment",
+          parent = err
+        )
+      }
+    )
+  }
+}
+
+appUsesPython <- function(quartoInfo) {
```

Comment:
Maybe this should still live in `bundle.R` since that's where it's used?

Reply:
I'm OK with it here.